docs/gettinginvolved/infrastructure/release-schedule-versioning.rst

```diff
@@ -77,7 +77,7 @@ Documentation releases will be made when documentation for a major or minor mark
 
 Documentation releases will be planned and scheduled by ClusterHQ's product team, in consultation with the marketing and engineering teams.
 
-The version of a documentation will have the version of the corresponding marketing release, with a ``+docX`` release, where ``X`` starts at ``1`` and is incremented for each documentation release.
+The version of a documentation will have the version of the corresponding marketing release, with a ``.postX`` release, where ``X`` starts at ``1`` and is incremented for each documentation release.
```

Comment:
I think if somebody needs to look at historical versions, they can look at the documentation from a corresponding release.